source/release-notes/1.4.txt

```diff
@@ -0,0 +1,32 @@
+.. _c2c-release-notes-1.4:
+
+===================================
+Release Notes for ``mongosync`` 1.4
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.4 Released <<TBD>>, 2023
+
+New Features
+-------------
+
+Set Cluster Workload Level
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.4.0, {+c2c-product-name+} allows you to set the cluster
```

Comment:
The source updated while I was writing my last comment, I'm not sure if it went out or not. The current text is good. 

See above for syncing - synchronizing

Reply:
Ok. Changed sync word. Also, it has to be "In 1.4.0" rather than "Starting in 1.4.0" because 6.0.5 also is the min version for 1.3.0.